### PR TITLE
add communication channels to other pages

### LINKED ISSUE #389: Update contact info
Noticed a few links we may want to make more readily available:

- Point to pulp-dev or pulp matrix rooms here (where we point to pulp-dev mailing list) https://pulpproject.org/get_involved/
- Make mention of pulp-list a link at bottom of https://pulpproject.org/installation-introduction/ (and maybe point to matrix pulp room)
- Perhaps we point to discourse as well?

### PR DESCRIPTION
**Pull request by melcorr**
fixes #389

### CODE REVIEW

File: installation-introduction.md [hunk 1]
```diff
@@ -35,3 +35,7 @@ There is a [proof of concept](https://raw.githubusercontent.com/pulp/pulp-operat
 ## Do you need something else?
 
 If you are blocked and don't find an option that you need, please write to `pulp-list@redhat.com` and let us know what problem you encountered or what scenario you're missing.
+
+We welcome you to open a discussion on our [Pulp Community Discourse](https://discourse.pulpproject.org). Feel free to introduce yourself, what you're trying to achieve, where you ran into problems or didn't understand something.
+
+You can also find us on [**pulp** on Matrix](https://matrix.to/#/!HWvLQmBGV for user support.
```
Comment (daviddavis) [inline installation-introduction.md]: seems like `)` is missing?

Comment (melcorr) [reply installation-introduction.md]: it was much worse than that - half the link was missing. 
Some seriously rogue backspacing must have happened without me realising. 
Thank you!

File: installation-introduction.md [hunk 2]
```diff
@@ -35,3 +35,7 @@ There is a [proof of concept](https://raw.githubusercontent.com/pulp/pulp-operat
 ## Do you need something else?
 
 If you are blocked and don't find an option that you need, please write to `pulp-list@redhat.com` and let us know what problem you encountered or what scenario you're missing.
+
+We welcome you to open a discussion on our [Pulp Community Discourse](https://discourse.pulpproject.org). Feel free to introduce yourself, what you're trying to achieve, where you ran into problems or didn't understand something.
+
+You can also find us on [**pulp** on Matrix](https://matrix.to/#/!HWvLQmBGVPfJfTQBAu:matrix.org?via=libera.chat&via=matrix.org&via=ctrl-c.liu.se) for user support.
```
Comment (mdellweg) [inline installation-introduction.md]: Wouldn't that be enough for the link?
```suggestion
You can also find us on [**pulp** on Matrix](https://matrix.to/#/!HWvLQmBGVPfJfTQBAu:matrix.org) for user support.
```

Comment (melcorr) [reply installation-introduction.md]: yep, that worked.

Comment (mdellweg) [reply installation-introduction.md]: Querystrings like that make me nervous...

File: get_involved.md [hunk 3]
```diff
@@ -17,6 +17,22 @@ list.
...
+
+## Chat to us
+
+The Pulp community's primary communication channel is Matrix.
+
+We have corresponding Libera.Chat rooms for every Matrix room, so if you use IRC exclusively, you can find us there too.
+
+* Join [**pulp** on Matrix](https://matrix.to/#/!HWvLQmBGVPfJfTQBAu:matrix.org?via=libera.chat&via=matrix.org&via=ctrl-c.liu.se) for User support.
+
+* Join [**#pulp-dev** on Matrix](https://matrix.to/#/!aVApiNMtnstWbwDcVU:matrix.org?via=libera.chat&via=matrix.org&via=ctrl-c.liu.se) for Developer discussion.
```
Comment (mdellweg) [inline get_involved.md]:
```suggestion
* Join [**pulp** on Matrix](https://matrix.to/#/!HWvLQmBGVPfJfTQBAu:matrix.org) for User support.

* Join [**#pulp-dev** on Matrix](https://matrix.to/#/!aVApiNMtnstWbwDcVU:matrix.org) for Developer discussion.
```

Comment (melcorr) [reply get_involved.md]: Would you believe I went to great efforts to secure https://matrix.to/#/#pulp-dev:matrix.org and https://matrix.to/#/#pulp:matrix.org and didn't even use them here! 
Thanks. Have updated.

**Review by mdellweg — APPROVED** [review]

**Comment by melcorr** [comment]
Thanks for the reviews!

**Pull request merged**